crates/shrs_core/src/keybinding.rs

```diff
@@ -27,15 +28,17 @@ pub type Binding = (KeyCode, KeyModifiers);
 #[macro_export]
 macro_rules! keybindings {
     // TODO temp hacky macro
-    (|$sh:ident, $ctx:ident, $rt:ident| $($binding:expr => $func:block),* $(,)*) => {{
+    (|$sh:ident, $ctx:ident, $rt:ident| $($binding:expr => ($func:block, $desc:expr)),* $(,)*) => {{
```

Comment:
adding description is a good idea. perhaps switching the order of the func and description would be more appealing to the eyes though.